ml-proto/spec/kernel.ml

```diff
@@ -119,16 +119,18 @@ and func' =
 type memory = memory' Source.phrase
 and memory' =
 {
-  initial : Memory.size;
+  min : Memory.size;
```

Comment:
I think `initial` was the better name for this, since `min` doesn't convey that this is also the initial size of memory.  E.g., in the future, maybe we add a `shrink_memory` operator and then requiring declaring a separate `min` size that was <= `initial`.